src/main/java/com/minecrafttas/common/mixin/MixinNetHandlerPlayClient.java

```diff
@@ -0,0 +1,39 @@
+package com.minecrafttas.common.mixin;
+
+import java.net.ConnectException;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import com.minecrafttas.common.events.client.player.EventOtherPlayerJoinedClientSide;
+import com.minecrafttas.common.events.client.player.EventPlayerJoinedClientSide;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.network.NetHandlerPlayClient;
+import net.minecraft.network.play.server.SPacketPlayerListItem;
+import net.minecraft.network.play.server.SPacketPlayerListItem.Action;
+
+@Mixin(NetHandlerPlayClient.class)
+public class MixinNetHandlerPlayClient {
+	@Shadow
+	private Minecraft gameController;
+
+	@Inject(method = "handleJoinGame", at = @At(value = "RETURN"))
+	public void clientJoinServerEvent(CallbackInfo ci) throws ConnectException {
+		EventPlayerJoinedClientSide.firePlayerJoinedClientSide(gameController.player);
+	}
+
+	//Github Workflows can't process this for some reason...
+	
```

Comment:
hmm? if "github workflows" == compiler error && is mixin error, then compilation won't work either because the mappings are incorrect/missing

Reply:
I just had no idea, why and I already forgot the error... Maybe it was forge gradle acting up or whatever, but here I forgot to remove the comment

Reply:
documentation pr ig?